ops-bedrock/docker-compose.yml

```diff
@@ -12,6 +12,7 @@ volumes:
 
 services:
   l1:
```

Comment:
I'm not so sure we want these images specified in the compose? The compose file is used as part of the CI tests, so I think we need it to be referencing the local images, or, we need to modify the way they're tagged?

Reply:
ok, so I remove the tag in docker-compose and use docker with Dockerfile to build images with the right tag.